tests/js/spec/components/assistant/guideAnchor.spec.jsx

```diff
@@ -6,29 +6,26 @@ import GuideActions from 'app/actions/guideActions';
 import ConfigStore from 'app/stores/configStore';
 
 describe('GuideAnchor', function() {
-  const guides = {
-    guide1: {
-      id: 1,
-      required_targets: [],
-      steps: [
-        {message: 'abc', target: 'target 1', title: 'title 1'},
-        {message: 'xyz', target: 'target 2', title: 'title 2'},
-      ],
+  let wrapper1, wrapper2;
+  const serverGuide = [
+    {
+      guide: 'issue_details',
```

Comment:
Should this be dash cased?

Reply:
this is the string guide key from the backend

Reply:
doy I see